integration-tests/jest-reporter.js

```diff
@@ -0,0 +1,55 @@
+const { createWriteStream, existsSync, mkdirSync } = require('fs')
+const path = require('path')
+
+
+module.exports = class JestReporter {
+  constructor(_globalConfig, _options) {
+    this.logDir = process.env.SCREENSHOT_DIR || 'failures'
```

Comment:
I chose this dir because I wouldn't have to update CircleCI `config.yml`. 
How about `/results` dir? I'd also need a new env variable like RESULTS_DIR and update `config.yml`. Any other suggestion?

Reply:
I think results is better than failures. It looks like everything in ci is currently going to failure_screenshots, which is confusing.

I would be awesome to have some clear way to pick out resources related to failed tests vs. succeeded tests.

![image](https://user-images.githubusercontent.com/484484/165978479-79b7e925-e053-4d01-8ddf-893499943ad6.png)